lib/watir_drops/page_object.rb

```diff
@@ -18,12 +22,26 @@ def page_url
         end
       end
 
+
+      def page_title(required: true)
+        @require_title = true if required
```

Comment:
What is `#page_title` going to be used for other than to verify `#on_page?` If there isn't a use for it, I don't think we need the `required` parameter for it.